installer/ldap/install.sh

```diff
@@ -19,19 +19,16 @@
 . ../utils/wait_for_pod.sh
 RELEASE_NAME="$1-openldap"
 export LDAP_DOMAIN_NAME="$RELEASE_NAME.default"
-DIR=`pwd`
-cd $HELM_TEMP_DIR/ldap-subchart/
+cd $HELM_TEMP_DIR/ldap-subchart/certs
 header "Installing LDAP"
-cd certs
 ./genereate_ldap_certs.sh
-cd ..
-if [ "$MGT_API_AUTHORIZATION" == "true" ]; then
-helm install --name $RELEASE_NAME -f ./customLdifFiles.yaml .
+cd -
```

Comment:
@joannanosek Here we are returning to previous dir.

Reply:
Hmm so we have inconsistency, because inside some install scripts we have `cd -`, in some not, but we always have returning in main after every script 😃 but ok, I'm ok to leave it as it is for now